tests/browser/remote_cdp.py

```diff
@@ -0,0 +1,28 @@
+import asyncio
+
+from playwright.async_api import async_playwright
+
+
+async def test_remote_cdp_connection():
+	# Remote debugging URL - replace with actual if needed in local/dev
+	cdp_url = 'http://localhost:9222'  # This port must be open in CI if testing real CDP
```

Comment:
just launch the browser in a fixture like we do in `test_tab_management.py` and `test_controller.py`